### PR TITLE
[MWPW-132768] Remove parse.js logic from FEDS checks

### PR DESCRIPTION
**Pull request by chadsunvice**
PR related to [MWPW-132768](https://jira.corp.adobe.com/browse/MWPW-132768): Remove parse.js logic from FEDS checks

**Acceptance Criteria**:

* update **FEDS checks** logic to not use **parse.js** library
* create additional logic supporting the **FEDS checks** where required
* test files affected:
  * userprofile.test.js
  * consent.test.js 
  * header.test.js
  * search.test.js

* linting/refactoring some Milo checks

### CODE REVIEW

**Review by amauch-adobe — APPROVED** [review]
Approved, looks good to me.  Though I think @skumar09 should review this before going in since most of the block.test.js milo test files were done by him.  Also, it would be good to have the CI run the tests at least for Feds since they were swapped out from using the Parse.js file.

**Comment by chadsunvice** [comment]
@amauch-adobe, @skumar09, please re-review.

- I removed ` › ` with `, ` as mentioned in the previous reviews (PR from @iamdanchiv account)
- talked with Narcis and as Milo is using ES6, we want to pursue using only `imports`, also file foot-print is smaller (1 import, vs. 1 require + 1 constant variable assignment)
- now only importing `{ features }`, which uses the exact information required
- ran checks locally and on remote and nothing was failing, if any concerns, please reply with failing run to re-assess
- rest of the issues we can tackle in a future PR if required, for now let's merge these as is

**Review by amauch-adobe — APPROVED** [review]
Approved. Looks great!

**Comment by amauch-adobe** [comment]
@chadsunvice make sure to make the camelCase adjustments that @skumar09 mentioned.

File: tests/feds/consent.test.js [hunk 1]
```diff
@@ -3,13 +3,11 @@
 /* eslint-disable import/extensions */
 import { expect, test } from '@playwright/test';
 import { FedsConsent } from '../../selectors/feds/feds.consent.page.js';
-import * as ConsentSpec from '../../features/feds/consent.spec.js';
-
-const { features } = ConsentSpec;
+import { features } from '../../features/feds/consent.spec.js';
```
[the remark this answers is not in the document]
Comment (chadsunvice) [reply tests/feds/consent.test.js:6]: Great, thanks @amauch-adobe!

**Review by JackySun9 — APPROVED** [review]

**Pull request merged**